### DISCUSSION TITLE
sidebar目录(dir)名首字母英文大写调整

### DISCUSSION THREAD

**Discussion by raysun0501** [vuepress-theme-hope/vuepress-theme-hope#3317] [Q&A] [answered]
场景描述：sidebar的目录(dir)名首个字符为字母时，会无视原字母大小写，统一将字符转为大写英文字母，请问这个有方法可以配置吗？

**Comment by raysun0501** [comment]
补充：目录下无READEME.md文件。

**Comment by Mister-Hope** [comment]
有什么场景是首字母不能大写的么？

**Comment by Mister-Hope** [comment, marked as answer]
Hi there, to automatically generate readable titles, we are making some convertions, if you don't like them, you can ALWAYS set a dir title yourself by adding the following frontmatter:


```md
---
dir:
  title: Anything you want
---

File content...
```

We can not cancel this, see the following test example:

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/7dbbc047-e3bf-442f-9aab-f266934af637)
